docs/data-sources/federation_federated_blocks.md

```diff
@@ -0,0 +1,79 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_federation_federated_blocks Data Source - terraform-provider-bloxone"
+subcategory: ""
+description: |-
```

Comment:
description is missing for the new services

Reply:
done